public/css/style.css

```diff
@@ -1,10 +1,19 @@
+body {
+    background-image: url("../assets/wood-background.png");
```

Comment:
I do not think this pic match our theme. Maybe we can find a better one? lol

Reply:
Ok. I'll put a temporary plain background for now. We can change it before the presentation.